test/RootChainTest.js

```diff
@@ -36,53 +40,86 @@ contract("RootChain", ([owner, nonOwner, priorityQueueAddr]) => {
-            const expectedExitable_at = (await latestTime()) + duration.weeks(2);
+        it("should be equal utxoPos and exitableAt ", async () => {
+            const expectedExitableAt = (await latestTime()) + duration.weeks(2);
 
-            await this.rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum);
-            const [utxo_pos, exitable_at] = await this.rootChain.getNextExit(ZERO_ADDRESS);
+            await rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum);
+            const [utxoPos, exitableAt] = await rootChain.getNextExit(ZERO_ADDRESS);
 
-            utxo_pos.should.be.bignumber.equal(this.expectedUtxoPos);
-            exitable_at.should.be.bignumber.equal(expectedExitable_at);
-            this.rootChain.getExit(utxo_pos).to.have.ordered.members([owner, ZERO_ADDRESS, depositAmount])
+            exitableAt.should.be.bignumber.equal(expectedExitableAt);
+            utxoPos.should.be.bignumber.equal(this.expectedUtxoPos);
 
+            const [expectedOwner, token, amount] = await rootChain.getExit(utxoPos);
+            expectedOwner.should.equal = owner;
+            token.should.equal = ZERO_ADDRESS;
+            amount.should.equal = depositAmount;
         });
 
         it("should fail if same deposit is exited twice", async () => {
-            await this.rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum);
-            await expectThrow(this.rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum), EVMRevert);
+            await rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum);
+            await expectThrow(rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum), EVMRevert);
         });
 
         it("should fail if transaction sender is not the depositor", async () => {
-            await expectThrow(this.rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum, { from: nonOwner }), EVMRevert);
+            await expectThrow(rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum, { from: nonOwner }), EVMRevert);
         });
 
-        it("should fail if utxo_pos is worng", async () => {
-            await expectThrow(this.rootChain.startDepositExit(this.expectedUtxoPos * 2, ZERO_ADDRESS, depositAmountNum), EVMRevert);
+        it("should fail if utxoPos is worng", async () => {
+            await expectThrow(rootChain.startDepositExit(this.expectedUtxoPos * 2, ZERO_ADDRESS, depositAmountNum), EVMRevert);
         });
 
-        it("should fail if value given is not equal to deposited value", async () => {
-            await expectThrow(this.rootChain.startDepositExit(this.expectedUtxoPos, ZERO_ADDRESS, depositAmountNum + 1), EVMRevert);
-        })
+        // Here will fail if the 3rd argument is "depositAmountNum + m (-8 <= m <= 8)" because of convert(_amount, "bytes32") in L.260
```

Comment:
The reason is that `depositAmountNum` is Number object(not BigNumber object). (https://github.com/LayerXcom/plasma-mvp-vyper/pull/36/files#diff-74b776d6b43641d23f71b67d44f749afR23)

Number objects safely represent  numbers between -(2^53 - 1) and 2^53 - 1.
Because `0.1 ether = 10^17 wei > 2^53 - 1` , `depositAmountNum ± n(<8)` is equals to `depositAmountNum` .

ref. https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

Reply:
so it is not about 'convert()' .